src/tests/system/tests/test_default_debug_level.py

```diff
@@ -0,0 +1,196 @@
+"""
+Automation for default debug level
+
+:requirement: SSSD - Default debug level
+"""
+
+from __future__ import annotations
+
+import re
+
+import pytest
+from pytest_mh.ssh import SSHProcessError
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopology, KnownTopologyGroup
+
+
+@pytest.mark.topology(KnownTopology.Client)
+def test_default_debug_level__check(client: Client):
+    """
+    :title: Check default debug level when sssd started successfully
+    :setup:
+        1. Clear logs and cache
+        2. Start SSSD with default debug level
+    :steps:
+        1. Check log files
+    :expectedresults:
+        1. "Starting with debug level = 0x0070" is in each file
+            Log message with number "0x3f7c0" is stored, if log contains more than one line
+
+    :customerscenario: False
+    """
+    client.sssd.common.local()
+    client.sssd.default_domain = "local"
+
+    client.sssd.clear(db=True, memcache=True, logs=True, config=False)
+    client.sssd.start(debug_level=None)
+
+    for log_filename in ["sssd", "sssd_local", "sssd_nss", "sssd_pam"]:
+        log_str = client.fs.read(f"/var/log/sssd/{log_filename}.log")
+        check = False
+
+        starting = re.compile(r"Starting with debug level = 0x0070")
+        assert starting.search(log_str) is not None, f"Log file has wrong format: {log_str}"
+
+        log_lines = log_str.split("\n")
+        if len(log_lines) <= 1:
+            continue
+
+        for line in log_lines:
+            searched = re.compile(r"\(0x\w+\)").search(line)
+            if searched is not None and searched.group() == "(0x3f7c0)":
+                check = True
+                break
+        if not check:
+            assert False, "Wrong format of log file"
```

Comment:
Merge these two lines in one, making the assertion easier to understand

Reply:
Yes, of course